packages/rs-dapi-client/src/platform/broadcast_state_transition.rs

```diff
@@ -0,0 +1,46 @@
+//! `BroadcastStateTransition` request.
+
+use dapi_grpc::platform::v0::{self as platform_proto};
+use drive::dpp::state_transition::StateTransition;
+
+use super::IncompleteMessage;
+use crate::{transport::TransportRequest, DapiRequest, Settings};
+
+/// DAPI request to suggest a [StateTransition] to the network.
+#[derive(Debug)]
+pub struct BroadcastStateTransition {
+    /// State transition to broadcast
+    pub state_transition: StateTransition,
+}
+
+/// Response for [BroadcastStateTransition].
+/// Cannot carry any info as this process is not synchronous to see any
+/// results immediately except [StateTransition] was successfully broadcasted.
+/// For actual results [WaitForStateTransitionResult](crate::platform::WaitForStateTransitionResult) is used.
+#[derive(Debug)]
+pub struct BroadcastStateTransitionResponse {}
+
+impl DapiRequest for BroadcastStateTransition {
+    type DapiResponse = BroadcastStateTransitionResponse;
+
+    const SETTINGS_OVERRIDES: Settings = Settings::default();
+
+    type Error = IncompleteMessage;
+
+    type TransportRequest = platform_proto::BroadcastStateTransitionRequest;
+
+    fn to_transport_request(&self) -> Self::TransportRequest {
+        let mut cbor_state_transition = Vec::new();
+        ciborium::into_writer(&self.state_transition, &mut cbor_state_transition)
```

Comment:
We would use it if we wanted to go to `ciborium::Value`, because `serialize` is generic over it's output, here we have bytes so I'd prefer go straight to the point

Reply:
Yeah, let's keep it for now since it's supported on the Drive side.